Controllers/AttendanceController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using WsuTriangle.AttendanceTracker.Models;
+
+namespace WsuTriangle.AttendanceTracker.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AttendanceController(AppDbContext db) : ControllerBase
+{
+    [HttpPost]
+    // [Authorize(Roles = "Member")] // will figure out later
+    public async Task MarkAttendance(int eventId, int userId)
+    {
+        // var userId = GetUserIdFromToken() // Extract user ID from JWT // will figure out later
+        var attendance = new Attendance
```

Comment:
Some validation needs to be done here. What if an event doesn't exist? What if a user doesn't exist?

We can make use of Entity Framework here:

```
var user = db.Users.Where(x => x.UserId == userId).FirstOrDefault();
if (user is null) return; // there is a way to mark this as a bad request I forget though, try searching IActionResult?

var event = db.Events.Where(x => x.EventId == evnetId).FirstOrDefault();
if (event is null) return;

// using those navigational properties I added in Models!
event.Attendances.Add(new Attendance blah blah)

await db.SaveChangesAsync(); // entity framework magically creates the attendance and adds it to the event! its beautiful
```